src/Microsoft.DncEng.SecretManager/Readme.md

```diff
@@ -0,0 +1,196 @@
+# The .NET Core Engineering Secret Manager
+This tool is used to track and rotate the secrets in one or more key vaults, and to validate the usages of the same. This is done through a yaml manifest file for each vault, and json settings files for each service.
```

Comment:
> This is done through a yaml manifest file for each vault, and json settings files for each service.

I see the vault manifest described below but I don't see anything about the service settings file. Seems like a missing section?

Reply:
I didn't add that, I will do so.